recipes-containers/softwarecontainer/files/io.qt.ApplicationManager.Application.json

```diff
@@ -35,6 +35,38 @@
             }
           ]
         },
+        {
+          "id": "file",
+          "config": [
+            {
+              "path-host": "/run",
+              "path-container": "/run",
+              "read-only": false
+            },
+            {
+              "path-host": "/etc/ssl",
+              "path-container": "/etc/ssl",
+              "read-only": false
+            }
+          ]
+        },
+        {
+          "id":"network",
+          "config": [
+            {
+              "direction": "OUTGOING",
+              "allow": [
+                { "host": "*" }
```

Comment:
wouldn't it be better to allow traffic to map servers only?

Reply:
I missed some change here. I will update this.

Reply:
I tried using api.mapbox.com as hostname but that it is dropping some traffic resulting in failing to render map correctly. Probably map application is accessing something more than map server. So will keep the host as *

Reply:
or use Wireshark to check what hosts it is accessing...

Reply:
Have you checked https://docs.mapbox.com/help/troubleshooting/firewalls/ ? Add the list of domains and see if it works.

Reply:
Thanks @JEderonn I will try this

Reply:
Whitelisting these domains doesn't work. There is still a lot of traffic being dropped and hence no proper rendering of Map. 
I suggest that for the coming release this week, we can allow all traffic to get the maps working. It would have been a good example to selectively white list the hosts, but that can be investigated later.

Reply:
Ok, I would then propose that you create a new story for it and refer to the issue # here for tracking.

Reply:
User story created 
https://luxproject.luxoft.com/jira/browse/OAI1209-585